From c471cbe03c049ed4a97708936342e4dd28606d22 Mon Sep 17 00:00:00 2001
From: ro <ro@138bc75d-0d04-0410-961f-82ee72b054a4>
Date: Mon, 28 Mar 2011 16:46:27 +0000
Subject: [PATCH 016/200] 	Backport from mainline:
 	2011-03-23  Rainer Orth  <ro@CeBiTec.Uni-Bielefeld.DE>

	PR testsuite/48251
	* g++.dg/guality/guality.exp: Disable on alpha*-dec-osf*.


git-svn-id: svn+ssh://gcc.gnu.org/svn/gcc/branches/gcc-4_6-branch@171616 138bc75d-0d04-0410-961f-82ee72b054a4

index 9a17850..749691d 100644
--- a/gcc/testsuite/g++.dg/guality/guality.exp
+++ b/gcc/testsuite/g++.dg/guality/guality.exp
@@ -4,7 +4,8 @@ load_lib g++-dg.exp
 load_lib gcc-gdb-test.exp
 
 # Disable on darwin until radr://7264615 is resolved.
-if { [istarget *-*-darwin*] } {
+# Disable on Tru64 UNIX (PR testsuite/48251).
+if { [istarget *-*-darwin*] || [istarget alpha*-dec-osf*] } {
   return
 }
 
-- 
1.7.0.4

